planck-cljs/src/planck/socket.cljs

```diff
@@ -0,0 +1,79 @@
+(ns planck.socket
+  "Planck socket functionality."
+  (:require
+   [cljs.spec.alpha :as s]))
+
+(s/def ::host string?)
+(s/def ::port integer?)
+(s/def ::data string?)                                      ; Maybe also byte arrays in the future?
+(s/def ::socket integer?)
+(s/def ::data-handler ifn?)
+(s/def ::accept-handler ifn?)
+(s/def ::opts (s/nilable map?))
+
+(defn connect
+  "Connects a TCP socket to a remote host/port. The connected socket reference
+  is returned. Data can be written to the socket using `write` and the socket
```

Comment:
I gather cljdoc generates internal links based on `[[<link>]]`? So `` `write` `` should be updated, too?

Reply:
Good catch. Should be addressed in the updated push.